.github/workflows/xp.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         with:
           version: v1.41.1
           working-directory: common
-          args: --timeout 3m --verbose
+          args: --timeout 3m --verbose --skip-files=testutils/mocks/ManagementClientInterface.go
```

Comment:
I've had to add this because the `typecheck` linter was claiming that the autogenerated `ClientInterface` class uses an unknown method `Called`, which in reality exists as a method of the promoted `mocks.Mock` field of the struct. This is similar to the issue/discussion raised [here](https://github.com/golangci/golangci-lint/discussions/2287) though there doesn't seem to be any conclusive findings/solutions to this issue.

```
testutils/mocks/ManagementClientInterface.go:30:12: _m.Called undefined (type *ClientInterface has no field or method Called) (typecheck)
	ret := _m.Called(_ca...)
```